addon/helpers/object.js

```diff
@@ -25,7 +21,7 @@ export default MicroState.extend({
 
   actions: {
     recompute(current, params, options) {
-      return options;
+      return assign({}, options);
```

Comment:
Ok, here is what's happening as I see it: When the helper is evaluated for the first time, compute receives a frozen hash. This hash is used to create a prototype object. Prototype object is passed into `handlebarsValueFor` where we attempt to copy values from the original frozen hash onto the prototype object. This fails because properties on the prototype can not be overwritten (cause prototype is frozen).

It seems that an object with a prototype that's frozen maintains frozen state for properties from the prototype.